src/Authoring/AuthoringSample/Program.cs

```diff
@@ -462,7 +460,7 @@ public bool ContainsKey(string key)
             return _dictionary.ContainsKey(key);
         }
 
-        public void CopyTo(KeyValuePair<string, BasicStruct>[] array, int arrayIndex)
+        public void CopyTo([System.Runtime.InteropServices.WindowsRuntime.ReadOnlyArray] KeyValuePair<string, BasicStruct>[] array, int arrayIndex)

```

Comment:
We might need to improve our diagnostics to detect whether the function belongs to a custom type mapping.  In this case, this function is part of a custom mapping interface which means this function as is won't be projected.  But it would be removed and a mapping of it would be projected.

Reply:
I've updated it to now look for IList, IDictionary, and IReadOnlyDictionary, and filter out these interfaces' methods and properties from those that get scanned